src/main/java/org/myrobotlab/service/Hd44780.java

```diff
@@ -296,6 +484,10 @@ public void setBackLight(boolean status) {
    *          c
    * 
    */
+  // Why are we exposing the writeRegister publicly here.
+  // This is a dangerous method to the HD44780 as anything written here can destabilize it.
+  // not yet removed but should be depreciated or removed if not being used.
+  @Deprecated // use lcdWriteCmd(byte cmd) or lcdWriteData(byte data)
   public void writeRegister(byte cmd) {

```

Comment:
its used internally so it shouldn't be deprecated unless its mis-used,
but you should change the public to private